tests/plugins/divarea/manual/detached.md

```diff
@@ -0,0 +1,37 @@
+@bender-tags: 4.17.0, bug, 4462, divarea
```

Comment:
We use only the first three tags: version, issue type and issue number. It happened in the past we were adding here some other info like plugin name (I did it myself 🙈 ), but it doesn't bring any more value since the tests are already grouped by plugins. The only situation I can think of where it could be useful are integration tests between different plugins, but this is not the case here.

Same for all other tests.